OMEdit/OMEditLIB/Simulation/SimulationOutputWidget.cpp

```diff
@@ -575,6 +583,122 @@ void SimulationOutputWidget::compileModel()
 }
 
 
+/*!
+ * \brief SimulationOutputWidget::runPostCompilation
+ * Runs the post compilation command after the compilation of the model.
+ */
+void SimulationOutputWidget::runPostCompilation()
+{
+  const QString postCompilationCommand = OptionsDialog::instance()->getSimulationPage()->getPostCompilationCommand();
+  if (postCompilationCommand.trimmed().size())
+  {
+    mpPostCompilationProcess = new QProcess;
+    mpPostCompilationProcess->setWorkingDirectory(mSimulationOptions.getWorkingDirectory());
+    connect(mpPostCompilationProcess, SIGNAL(started()), SLOT(postCompilationProcessStarted()));
+    connect(mpPostCompilationProcess, SIGNAL(readyReadStandardOutput()), SLOT(readPostCompilationStandardOutput()));
+    connect(mpPostCompilationProcess, SIGNAL(readyReadStandardError()), SLOT(readPostCompilationStandardError()));
+  #if (QT_VERSION >= QT_VERSION_CHECK(5, 6, 0))
+    connect(mpPostCompilationProcess, SIGNAL(errorOccurred(QProcess::ProcessError)), SLOT(postCompilationProcessError(QProcess::ProcessError)));
+  #else
+    connect(mpPostCompilationProcess, SIGNAL(error(QProcess::ProcessError)), SLOT(postCompilationProcessError(QProcess::ProcessError)));
+  #endif
+    connect(mpPostCompilationProcess, SIGNAL(finished(int, QProcess::ExitStatus)), SLOT(postCompilationProcessFinished(int, QProcess::ExitStatus)));
+    writeCompilationOutput(QString("%1\n").arg(postCompilationCommand), Qt::black);
+    mpPostCompilationProcess->start(postCompilationCommand);
```

Comment:
`start` is deprecated. You need to split the command and arguments. See how `Tools->Options->General->Terminal Command` and `Tools->Options->General->Terminal Command Arguments` are done.

Reply:
this is not enough; the naive `split(" ")` wont work for quotes eg `sh -c "ls && ls"`
we need QProcess::splitCommand and have no choice but to continue to use start(QString) for qt<5.15 as splitCommand is not available there

Reply:
I agree with you.